tools/jit/gen_jit_dispatch.py

```diff
@@ -443,7 +443,18 @@ def format_arg(arg):
     else:
         ret_list = '({})'.format(', '.join(jit_type_of(r) for r in decl['returns']))
     name = decl['name'] if not is_out_variant(decl) else decl['name'][:-4]
-    return 'aten::{}({}) -> {}'.format(name, arg_list, ret_list)
+    ret_val = 'aten::{}({}) -> {}'.format(name, arg_list, ret_list)
+    # If matches_jit_signature has been specified the signature constructed from the
+    # declared attributes should match the raw string passed through. In the
+    # case of native_functions.yaml, func should match the generated signature,
+    # if matches_jit_signature is true. This is used to track and verify the alignment
+    # of native_function.yaml's function schema with that used in this parse.
+    if decl.get('matches_jit_signature'):
+        assert(ret_val.strip() == decl['schema_string'].strip()),\
```

Comment:
nit: do you need these `strips`?  certainly you don't on ret_val, because you just constructed it without extra spaces, what about schema_string?